core/dev/fhir/fhir-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: fhir
+  name: fhir
+spec:
+  ports:
+  - name: "3447"
```

Comment:
@bradsawadye Similar to my other comments about naming, please change this to `fhir-3447`

We want to know to what service this port belongs